tests/baselines/reference/isolated-declarations/auto-fixed/diff/declarationEmitBindingPatternWithReservedWord.d.ts.diff

```diff
@@ -0,0 +1,17 @@
+// [[Reason: #56992 Type printer preserves binding element aliases.]] ////

```

Comment:
Should this have been fixed by changes in main?

Reply:
Fixed, now it's just a source map difference